### DISCUSSION TITLE
qemu should no longer be the default for 'colima start'

### DISCUSSION THREAD

**Discussion by pastelmind** [abiosoft/colima#1201] [General]
I recently updated colima to v0.8.0 (using Homebrew) and couldn't start colima because qemu was no longer installed on my system. (I'm afraid I don't have the error message.) From #1188 I believe lima doesn't require qemu anymore?

Apparently, sometime in the past, I had followed the instructions on `README.md` and ran `colima delete && colima start`. This caused my colima to use qemu. I suppose anyone who follows the README like me would be confused because they, too, will be using qemu, which must be installed separately.

Maybe we could change the default for `--vm-type` to vz? Or perhaps detect whether qemu is available and fallback to vz otherwise.

Alternatively, we could add a section in the README to tell people to manually install qemu or use vz. A FAQ item would be nice, too.

**Comment by abiosoft** [comment]
Your issue is most likely due to having a template previously set with `colima template`.
QEMU is no longer the default when it is missing.

**Reply by pastelmind** [reply]
Good to know. I don't remember running `colima template` but maybe I had old templates? FYI I ran `colima delete`, deleted `~/.colima`, then rebooted my system, and now it works fine with vz.

**Reply by mlbiche** [reply]
Indeed, from a fresh installation of Colima 0.9.1 using Homebrew on MacOS Monterey (12.*), I am having an error about `qemu` missing on the first `colima start` run.

Running `colima delete` before doesn't change anything.

And the `--vm-type` flag is no longer recongnized (while it is mentionned in the README.md).

Creating a template configuration and setting `vmType` to `vz` do not fix the issue. The error is still that `qemu` is missing then.

Is that related to the fact I am running on a macOS version prior to macOS 13?

**Reply by abiosoft** [reply]
> Is that related to the fact I am running on a macOS version prior to macOS 13?

Yes

**Reply by mlbiche** [reply]
Thank you. So it is required to use `qemu` on macOS < 13?

**Reply by abiosoft** [reply]
> Thank you. So it is required to use qemu on macOS < 13?

Yes, sadly. VZ is only supported from macOS 13.

**Reply by mlbiche** [reply]
May I suggest an update to the README.md? I don't know the contributing policy but I would be pleased to clarify this point.

**Reply by abiosoft** [reply]
@mlbiche feel free to open a PR against the readme or FAQ.
